### PR TITLE
Meta tweaks

### PR DESCRIPTION
**Pull request by SamVerschueren**
Tweaked some things :tada:.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = (fn, opts) => {
 	const memoized = mem(fn, opts);
 
 	const ret = function () {
-		return memoized.apply(this, arguments).catch(err => {
+		return memoized.apply(this, arguments).catch(err => {		// eslint-disable-line prefer-rest-params
```
Comment (SamVerschueren) [inline index.js]: We could use spread if we dropped support for Node.js 4. Not sure on your standing for the `p-*` modules.

Comment (sindresorhus) [reply index.js]: Yes, feel free. I plan to drop Node.js 4 support in all my modules, just time...

File: .travis.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
```
Comment (sindresorhus) [inline .travis.yml]: Add `'10'`

**Comment by SamVerschueren** [comment]
Done

**Pull request merged**